File: lyg1071334926/UnitTestProject1/UnitTestProject1/UnitTest1.cs
using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace UnitTestProject1
{
    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestMethod1()
        {
            int a = 10, b = 15;
            int c = Calc.ADD(a, b);
            Assert.AreEqual(25, c);

        }
        public class Calc
        {
            public static int ADD(int a, int b)
            {
                return a + b;
            }
        }
    }
}

File: xxxy7280/Consoleapplication1/CalculatorUnitest/unittest1.cpp
#include "stdafx.h"
#include "CppUnitTest.h"
#include"../Calculator/Calculator.h"
using namespace Microsoft::VisualStudio::CppUnitTestFramework;

namespace CalculatorUnitest
{		
	TEST_CLASS(UnitTest1)
	{
	public:
		
		TEST_METHOD(TestMethod1)
		{
			Calculator* calc = new Calculator();
			string ret = calc->Solve("11+22");
			Assert::AreEqual(ret, (string)"11+22=33");
			// TODO: 在此输入测试代码
		}

	};
}